src/components/Filters/FilterDescription.tsx

```diff
@@ -17,12 +17,22 @@ const parseFilter = (filter: JSONSchema | null | undefined) => {
 };
 
 const FilterDescription = ({ filter, ...props }: FilterDescriptionProps) => {
-	const tagProps = filter.anyOf
+	const tagProps = !!filter.anyOf
```

Comment:
I guess the `!!` isn't necessary :)
```suggestion
	const tagProps = filter.anyOf
```